<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1956</width>
    <height>1044</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>2100</width>
      <height>1811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);

font: 12pt &quot;Times new roman&quot;;
color: white</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Graph">
     <property name="styleSheet">
      <string notr="true">selection-background-color:rgn(rgb(0, 0, 0));
font: 12pt &quot;Times new roman&quot;</string>
     </property>
     <attribute name="title">
      <string>Graph</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rear Left Leg</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RL_r_plot" native="true">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>60</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="RL_r_autoscale">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="RL_others_autoscale">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>730</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RL_others" native="true">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>540</y>
        <width>271</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RR_r_plot" native="true">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>60</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rear Right Leg</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="RR_r_autoscale">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FL_r_plot" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Front Right Leg</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FR_r_plot" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Front Left Leg</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="FL_r_autoscale">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="FR_r_autoscale">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="RL_th_autoscale">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>490</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RL_th_plot" native="true">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>300</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FL_th_plot" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="FL_th_autoscale">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>490</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FR_th_plot" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="FR_th_autoscale">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>490</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RR_th_plot" native="true">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>300</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="RR_th_autoscale">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>490</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="FR_others_autoscale">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>730</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FR_others" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>540</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="FL_others_autoscale">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>730</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="RR_others_autoscale">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>730</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="RR_others" native="true">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>540</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Select_data">
      <property name="geometry">
       <rect>
        <x>1830</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 45, 55);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Position</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Velocity</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times new roman</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;θ&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; vertical-align:sub;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt; Data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="FL_others" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times new roman</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;Others&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times new roman</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;r Data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>1710</x>
        <y>90</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Data (&lt;span style=&quot; font-style:italic;&quot;&gt;r,θ&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>1690</x>
        <y>140</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Data (&lt;span style=&quot; font-style:italic;&quot;&gt;Others&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Select_data_2">
      <property name="geometry">
       <rect>
        <x>1830</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 45, 55);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Current</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-tau_dhat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>External force</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Reference&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Actual&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="jnt_plot" native="true">
      <property name="geometry">
       <rect>
        <x>1420</x>
        <y>300</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="jnt_auto_scale">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>490</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="jnt_plot_2" native="true">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>540</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="jnt2_auto_scale">
      <property name="geometry">
       <rect>
        <x>1590</x>
        <y>730</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string>Auto Scale</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
